### ISSUE TITLE
Examples doesn't work

### ISSUE THREAD

**Issue by fobos** [rtfeldman/elm-css#67]
Hi @rtfeldman 

I tried to run `elm make HomepageView.elm` and got an error
```
I cannot find find module 'Html.Helpers'.

Module 'HomepageView' is trying to import it.

Potential problems could be:
  * Misspelled the module name
  * Need to add a source directory or new dependency to elm-package.json
```

It would be great you fix examples. I'm trying to use `elm-css` in one of my non-production app.

Thanks.

**Comment by rtfeldman** [comment]
Fixed in 199036c75267647d06382d405fc97ad1f1dc485a!

**Comment by fobos** [comment]
@rtfeldman Thanks! :+1:
